### DISCUSSION TITLE
Some Font Awesome 5 icons do not render

### DISCUSSION THREAD

**Discussion by siddhartha-lal** [requarks/wiki#3669] [Error / Bug Report]
**Describe the bug**
Some icons in the Font Awesome library do not render in the navigation section of the admin panel or the main wiki.

**To Reproduce**
1. Go the navigation section in the admin panel
2. Choose an icon like fas fa-house-user
3. Visit the wiki

**Expected behavior**
The icon should change according to the name specified

**Screenshots**
![icons_1](https://user-images.githubusercontent.com/56102539/80304587-0040f280-87dd-11ea-83d3-89ea9e21b667.png)

**Host Info (please complete the following information):**
 - OS: Ubuntu 18.04
 - Wiki.js version: 2.3.77
 - Database engine: MySQL (latest version)

**Additional context**
This only happens for some icons. Few other icons I tried: fas fa-caravan, fal fa-house-night

**Comment by justinacolmena** [comment]
I attempted to "upgrade" to Wiki.js 2.3.81 by downloading and unpacking the latest version, copying over `config.yml` and the `assets/` folder and as you can see, the icons (mostly "Material Design") are all messed up on my wiki. Odd because I simply installed the new version and re-used my old database, and nothing is wrong with the content itself: just the icons from the theme.

![wiki-js-icons-messed-up](https://user-images.githubusercontent.com/32494761/81265056-e4182f00-8fee-11ea-864d-38aefd8baa8a.png)

Any ideas?

**Comment by NGPixel** [comment]
@justinacolmena clear your cache

**Comment by MaxSliw** [comment]
@justinacolmena me too 🤷🏻‍♂️🤷🏻‍♂️ and yet I cleared the cache

**Comment by NGPixel** [comment]
Make sure you also clear any reverse proxy cache (e.g. Cloudflare, nginx, etc.)

**Comment by MaxSliw** [comment]
> Make sure you also clear any reverse proxy cache (e.g. Cloudflare, nginx, etc.)

You are the best ! Thx

**Comment by stilobique** [comment]
Hello,
i'm find this topic because i can rendering the Navigation section with all Font Awesome icon, however, with any research i'm find an option on Theme to activate a Icon Set.

Maybe this issue can be close @NGPixel .
![4FPSrXoKGW](https://user-images.githubusercontent.com/3066684/104518546-1eaf7e00-55f8-11eb-8355-7c7b067aa8a4.jpg)

**Comment by mlwilkerson** [comment]
Hi @siddhartha-lal, it looks like the `house-user` icon was introduced to Font Awesome 5 Free in version `5.13.0`.

<img width="1065" alt="image" src="https://user-images.githubusercontent.com/11575183/158689056-a42a27ef-a4a1-4594-9a1b-639045066dce.png">

However, as of Wiki.js 2.5.276, it looks like it loads FA Free 5.10.0. So any icons released after FA 5.10.0 won't be available.

<img width="784" alt="image" src="https://user-images.githubusercontent.com/11575183/158689745-d9a491fb-f028-4d17-95c0-e3a2f358f572.png">

@NGPixel as I [mentioned elsewhere](https://github.com/Requarks/wiki/discussions/2835#discussioncomment-2375670), I could offer suggestions, and maybe PRs, for how to integrate with Font Awesome that would enable using the latest available releases, FA Pro, Kits, and even Kit Icon Uploads. Our [GraphQL API](https://fontawesome.com/docs/apis/graphql/get-started) can enable any of this.
